<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Sphere Histogram</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="3">
     <widget class="QComboBox" name="comboBox">
      <property name="currentText">
       <string>Viridis</string>
      </property>
      <item>
       <property name="text">
        <string>Cividis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inferno</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Magma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plasma</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Turbo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Viridis</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBoxShowPoints">
      <property name="text">
       <string>Show Points</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QCheckBox" name="checkBoxShowSphere">
      <property name="text">
       <string>Show Sphere</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QCheckBox" name="checkBoxMirrorPoints">
      <property name="text">
       <string>Mirror Points</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="11,1">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="SphereWidget" name="sphereWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>7</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="labelMaxPointsPerTriangle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>255</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ColorBarWidget" name="colorBarWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>6</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMinPointsPerTriangle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionViridis">
   <property name="text">
    <string>Viridis</string>
   </property>
  </action>
  <action name="actionMagma">
   <property name="text">
    <string>Magma</string>
   </property>
  </action>
  <action name="actionInferno">
   <property name="text">
    <string>Inferno</string>
   </property>
  </action>
  <action name="actionPlasma">
   <property name="text">
    <string>Plasma</string>
   </property>
  </action>
  <action name="actionCividis">
   <property name="text">
    <string>Cividis</string>
   </property>
  </action>
  <action name="actionTurbo">
   <property name="text">
    <string>Turbo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SphereWidget</class>
   <extends>QWidget</extends>
   <header>spherewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorBarWidget</class>
   <extends>QWidget</extends>
   <header>colorbarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
